src/PresentationalComponents/ImpactBadge/ImpactBadge.js

```diff
@@ -0,0 +1,85 @@
+import { QuestionIcon, SecurityIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React from 'react';
+
+class ImpactBadge extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.colorList = {
```

Comment:
Can you move this either above the component or to special const file? So we don't recreate these values everytime component is created?

Reply:
Where I should to place this special const file and name it (including class name) in file structure, please? Is there any rule for it? Or existing example?

Reply:
I don't think so, but you can create new file next to this one called `consts.js` and export these variables there.